CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+All notable changes to OpenVM will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Breaking Changes
+
+#### CLI
+
+- **New `init` command**: Added `cargo openvm init` command for creating new OpenVM packages with proper project structure and dependencies. This command initializes a new Rust project with OpenVM configuration.
+
+- **New `commit` command**: Added `cargo openvm commit` command for viewing the Bn254 commit of an OpenVM executable. This command generates and displays commitment information for built executables.
+
+- **Setup command renamed**: The `EvmProvingSetupCmd` has been renamed to `SetupCmd`. Scripts and automation that reference the old command structure will need to be updated.
```

Comment:
don't mention the code-level change, only what the actual change to the `cargo openvm` cmdline command is -- in this case I think there was no change